app/assets/javascripts/modules/base-target.js

```diff
@@ -0,0 +1,26 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function BaseTarget () { }
+
+  BaseTarget.prototype.start = function () {
+    if (this.pageIsInIframe()) {
+      this.setBaseTarget()
+    }
+  }
+
+  BaseTarget.prototype.setBaseTarget = function () {
+    var base = document.createElement('base')
+    base.target = '_blank'
```

Comment:
I suggest adding `rel="noopener noreferrer"` to the links for [security reasons](https://medium.com/sedeo/how-to-fix-target-blank-a-security-and-performance-issue-in-web-pages-2118eba1ce2f).

Reply:
I agree with the suggestion, but I'm not sure the `rel` attribute works on a base element and I would avoid querying all the links within the page because of the computational complexity.

Reply:
Agreed 👍 thanks